uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Mail/Message.php

```diff
@@ -140,10 +140,13 @@ public function getLeaveContactID() {
    * @return string
    */
   private function getLeaveRequestURL($contactID) {
-    $leaveUrl = CRM_Utils_System::url('my-leave', [], true);
+    $queryString = '?leave-request-id=' . $this->leaveRequest->id;
+    $leaveUrl = CRM_Utils_System::url('my-leave#/my-leave/report' .
+      $queryString, [], true);
 
     if ($this->leaveRequest->contact_id != $contactID) {
-      $leaveUrl = CRM_Utils_System::url('manager-leave', [], true);
+      $leaveUrl = CRM_Utils_System::url('manager-leave#/manager-leave/requests' .
```

Comment:
Looks good. Are the tests passing? 
I think you might need to update the tests here: https://github.com/civicrm/civihr/blob/f34867484a4acfaa989ef7af2c5fd2036c0a8aff/uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Mail/MessageTest.php#L162-L162